<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchesView</class>
 <widget class="QMainWindow" name="BatchesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить партию</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Icons.qrc">
    <normaloff>:/Icons/MainIcon/Resources/MainIcons/free-icon-book-4341050.png</normaloff>:/Icons/MainIcon/Resources/MainIcons/free-icon-book-4341050.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Произведение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="work_box"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Печатный центр</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="print_box"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Колличество копий</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="count_spin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_batch">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
